_studio/mfx_lib/shared/src/libmfxsw_encode.cpp

```diff
@@ -80,20 +83,16 @@ struct CodecKey {
 };
 
 struct Handlers {
-    typedef std::function<VideoENCODE*(VideoCORE* core, mfxCoreInterface* coreInt, mfxStatus *mfxRes)> CtorFunc;
-
     struct Funcs {
-        CtorFunc ctor;
+        std::function<VideoENCODE*(VideoCORE* core, mfxCoreInterface* coreInt, mfxStatus *mfxRes)> ctor;
```

Comment:
You have some strange playing in prev. patches around this: please, remove and use single variant right away.